packages/material-components-web/index.js

```diff
@@ -32,7 +32,6 @@ import * as floatingLabel from '@material/floating-label/index';
 import * as formField from '@material/form-field/index.ts';
 import * as gridList from '@material/grid-list/index';
 import * as iconButton from '@material/icon-button/index.ts';
-import * as iconToggle from '@material/icon-toggle/index';
```

Comment:
You forgot to remove `iconToggle` from exports below (shouldn't this break?)

Reply:
ah that probably came up in npm run build but not the dev server. Thanks!